bluez-async/src/lib.rs

```diff
@@ -52,6 +52,9 @@ use tokio::time::timeout;
 use uuid::Uuid;
 
 const DBUS_METHOD_CALL_TIMEOUT: Duration = Duration::from_secs(30);
+// in dbus C lib the max value is #define DBUS_TIMEOUT_INFINITE ((int) 0x7fffffff)
+// 0x7fffffff (the largest 32-bit signed integer) or INT32_MAX
+const DBUS_METHOD_CALL_MAX_TIMEOUT: Duration = Duration::from_secs(u32::MAX as u64);
```

Comment:
Should this be `i32::MAX`, according to your comment above?

Reply:
sure, fixed